AUTHORS

```diff
@@ -1,3 +1,4 @@
 Jerel Unruh <mail@unruhdesigns.com>
 Greg Aker <greg@gregaker.net>
+Adam Wróbel <https://adamwrobel.com>
```

Comment:
Thanks. I totally understand not wanting to put your email address. I don't like doing that either for the same reason you described. Thanks for adding your name.